.github/workflows/coverage.yml

```diff
@@ -37,7 +37,10 @@ jobs:
         run: npm install
 
       - name: Run the tests
-        run: npm test -- --coverage
+        run: npm run e2e:ci
+
+      - name: Covert test results
+        run: npx nyc report --reporter=json --reporter=html
 
```

Comment:
I had read that unit tests would be covered by just running this by itself, but it appears that you have to manually instrument your test files for this to work, and even then I'm not sure if it'll do what we want it to.
I did some research and it seems that the best thing to do is to run the test frameworks and then immediately upload the coverage report to CodeCov as it will automatically merge the reports together. I'll try this out and see if this does what we want it to.

Reply:
so it worked after all 😁